cves/CVE-2011-3944.yml

```diff
@@ -144,10 +144,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
```

Comment:
You can check out FFmpeg's code coverage tool too, it kind of implies if the unit tests they have are good or not http://coverage.ffmpeg.org/index.src_libavcodec_smacker.c.html